frontend/src/views/Room.jsx

```diff
@@ -210,7 +236,6 @@ function Room() {
       });
 
       newRoom.on('ParticipantJoined', async (p) => {
-        // console.log('someone joined', p);
         p.subscribe();
```

Comment:
Hey! This looks good! I was thinking, shouldn't we also stop subscribing to the participant if we don't display its camera? I **think** that even though we are not rendering the video element, the data stream is still being received since there's no limit set on webrtc.js.

Reply:
This is a good point. We should unsubscribe from the video to save bandwidth, re-sub again once we hear that person talking and show the video if so.

Reply:
I agree, I was just about to comment on that! We'll need to find the best way to get an event from the server that someone started speaking, so we can subscribe to their streams.

Also, this is a hybrid event: I don't think we should always be able to hear everyone in the call. Every user who's a host or speaker, yes. But we might want to think about having other users muted by default, and only be given the ability to share certain streams by a user with a higher power.

This is probably a whole other ticket, though 😁